docs/app_metrics_guide.adoc

```diff
@@ -0,0 +1,41 @@
+= Understanding & taking actions based on App Metrics
```

Comment:
= Analyzing  App usage using the metrics service
This guide shows how to use app metrics to provide rich visualizations regarding app usage that can then drive decisions, such as 'when will we stop supporting version x.y of app?'